scripts/e2e.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash -x
 
-source ${OKTA_HOME}/${REPO}/scripts/setup.sh
+source ${OKTA_HOME}/${REPO}/scripts/setup.sh "v14.18.0"
```

Comment:
Does it require node 14 to run the E2E?

Reply:
It works when I run it locally (on 12.22), it may not required node 14. I think `protractor` required it https://github.com/okta/okta-react/blob/master/scripts/e2e-samples.sh#L3